### PR TITLE
review of the DataFrames.jl tutorial

### PR DESCRIPTION
**Pull request by bkamins**
Clean up after #2717

### CODE REVIEW

File: docs/src/man/basics.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 **The tutorial section of the manual is still work in progress. Please report any questions or comments as issues in DataFrames.jl GitHub repository. Thank you!**
 
+## Setting up the environment
```
Comment (bkamins) [inline docs/src/man/basics.md:5]:
```suggestion
## Setting up the Environment
```

File: docs/src/man/basics.md [hunk 2]
```diff
@@ -777,28 +764,13 @@ julia> german[:, [:Age, :Sex]]
...
-     │ String  Int64
-─────┼───────────────
-   1 │ male       67
-   2 │ female     22
-   3 │ male       49
-   4 │ male       45
-   5 │ male       53
-```
-
 Pay attention that `german[!, [:Sex]]` and `german[:, [:Sex]]` return a data frame object,
```
Comment (pdeffebach) [inline docs/src/man/basics.md:767]:
```suggestion
Pay attention that `german[!, [:Sex]]` and `german[:, [:Sex]]` returns a data frame object,
```

File: docs/src/man/basics.md [hunk 3]
```diff
@@ -777,28 +764,13 @@ julia> german[:, [:Age, :Sex]]
...
-while `german[!, :Sex]` and `german[:, :Sex]` return a vector. In the first case, `[:Sex]` 
+while `german[!, :Sex]` and `german[:, :Sex]` return a vector. In the first case, `[:Sex]`
```
Comment (pdeffebach) [inline docs/src/man/basics.md:768]:
```suggestion
while `german[!, :Sex]` and `german[:, :Sex]` returns a vector. In the first case, `[:Sex]`
```

File: docs/src/man/basics.md [hunk 4]
```diff
@@ -1100,18 +1014,19 @@ julia> df1
...
+of rows of `df1` or as a special case if `df1` would not have any columns.
 
 ```jldoctest dataframe
 julia> df1.Age === val # no copy is performed
 true
 ```
 
-Below setting values of column `:Job` in rows `1:3` to values `[2, 4, 6]` *in-place* means write to an existing 
-column in a data frame.
+If select only some rows from a data frame the mutation is peformed in place, i.e.
```
Comment (pdeffebach) [inline docs/src/man/basics.md:1024]: Not sure what this sentence means.

**Review by pdeffebach — CHANGES_REQUESTED** [review]
Looks good! 

I couldn't compile the documentation, but reviewed this after eliminating the `jldoctest` modifiers and using a default markdown viewer.

File: docs/src/man/basics.md [hunk 5]
```diff
@@ -254,24 +173,24 @@ julia> DataFrame([1 0; 2 0], :auto)
...
 ```jldoctest dataframe
-julia> P = [1 2 4 5; 15 58 69 41; 23 21 26 69]
+julia> mat = [1 2 4 5; 15 58 69 41; 23 21 26 69]
 3×4 Matrix{Int64}:
   1   2   4   5
  15  58  69  41
  23  21  26  69
 
-julia> H = string.('a':'d')
+julia> nms = string.('a':'d')
```
Comment (bkamins) [inline docs/src/man/basics.md:186]:
```suggestion
julia> nms = ["a", "b", "c", "d"]
```

[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/basics.md:186]: good point

File: docs/src/man/basics.md [hunk 6]
```diff
@@ -284,17 +203,29 @@ julia> DataFrame(P, H)
 To move forward with the tutorial you need to install the CSV.jl package in your environment.
```
[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/basics.md:203]: You mean about installation of CSV.jl? I think it is not strictly necessary, but the tutorial is aimed as people who start working with Julia, so I think it is no problem to keep it.

[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/basics.md:203]: indeed - removed.

File: docs/src/man/basics.md [hunk 7]
```diff
@@ -284,17 +203,29 @@ julia> DataFrame(P, H)
...
+instructions:
+
 ```julia
 julia> using Pkg
 julia> Pkg.add("CSV")
 ```
 
-Make sure you have CSV.jl in a version that is at least 1.0.
-
 Now, we will explore how to load a CSV file into a `DataFrame`. Unlike Python's Pandas `read_csv`
```
[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/basics.md:223]: removed

File: docs/src/man/basics.md (hunk 4 above)
Comment (bkamins) [reply docs/src/man/basics.md:1024]: fixed. Essentially means to say that if row selector is not `!` then in-place operation is performed

File: docs/src/man/basics.md [hunk 8]
```diff
@@ -322,39 +253,55 @@ julia> german_ref = CSV.read(joinpath(dirname(pathof(DataFrames)),
...
-Now let's talk about the given code block:
+As you can see the data frame is wider and taller than the display width, so it
+got cropped and its 4 rightmost columns and middle 985 rows were not printed.
+Later in the tutorial we will discuss how to force Julia to show the whole
+data frame if we wanted so.
+
+Also observe that DataFrames.jl (unlike Python's Pandas) displays the data type of the column
+below its name. In our case, it is an `Int64`, or `String`.
+
+Before moving forward let us explain in detail the following code block:
```
[the remark this answers is not in the document]
Comment (bkamins) [reply docs/src/man/basics.md:266]: I have checked - it renders correctly.

**Comment by bkamins** [comment]
Thank you for the review. All comments are now hopefully addressed.

**Review by pdeffebach — APPROVED** [review]

File: docs/src/man/basics.md [hunk 9]
```diff
@@ -13,19 +15,22 @@ julia> Pkg.add("DataFrames")
...
 (@v1.6) pkg> add DataFrames
 ```
 
 If you want to make sure everything works as expected you can run the tests bundled
-with DataFrames.jl, but be warned that it will take more than 30 minutes.
+with DataFrames.jl, but be warned that it will take more than 30 minutes:
+
 ```julia
 julia> using Pkg
 julia> Pkg.test("DataFrames")
```
Comment (pdeffebach) [inline docs/src/man/basics.md:28]:
```suggestion
julia> Pkg.test("DataFrames") # Warning! This will take more than 30 minutes.
```

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**